internal/runners/commit/commit.go

```diff
@@ -78,6 +85,12 @@ func (c *Commit) Run() (rerr error) {
 		return errs.Wrap(err, "Could not get local build script")
 	}
 
+	for _, fc := range script.FunctionCalls("ingredient") {
+		if err := NewIngredientCall(c.prime, script, fc).Resolve(); err != nil {
+			return errs.Wrap(err, "Could not resolve ingredient")
+		}
+	}
```

Comment:
mm great points. I think for now since we're wanting to experiment with this I'll leave it as is, but if we stick with this approach then adding progress indication is something we'll likely want to iterate on.